pkg/controller/dnszone/awsactuator.go

```diff
@@ -433,22 +443,30 @@ func (a *AWSActuator) Delete() error {
 
 func (a *AWSActuator) deleteRecordSets(logger log.FieldLogger) error {
```

Comment:
Is this function still needed?

Would it be better to just have everything call `DeleteAWSRecordSets` directly?